editor/src/messages/portfolio/document/overlays/utility_types.rs

```diff
@@ -38,10 +38,10 @@ impl OverlayContext {
 			self.render_context.line_to(quad.0[i].x.round() - 0.5, quad.0[i].y.round() - 0.5);
 		}
 		if let Some(color_fill) = color_fill {
-			self.render_context.set_fill_style(&wasm_bindgen::JsValue::from_str(color_fill));
+			self.render_context.set_fill_style_str(color_fill);
```

Comment:
I just tested the build link and I'm noticing all our overlays, as set in this file, are no longer blue or whatever color— they're all the default black now. So I think some/all of these color setters aren't working, but I wouldn't be sure if that's due to the way we're calling them or an issue with the wasm-bindgen/web-sys/js-sys upstream code. Would you mind taking a look, please?

Reply:
You have to update your version of wasm-bindgen-cli (see the docs I updated here) @Keavon.

Reply:
`cargo install -f wasm-bindgen-cli@0.2.95`